constraint/pkg/client/client.go

```diff
@@ -622,7 +625,26 @@ func (c *Client) RemoveData(ctx context.Context, data interface{}) (*types.Respo
 // Review makes sure the provided object satisfies all stored constraints.
 // On error, the responses return value will still be populated so that
 // partial results can be analyzed.
-func (c *Client) Review(ctx context.Context, obj interface{}, opts ...drivers.QueryOpt) (*types.Responses, error) {
+func (c *Client) Review(ctx context.Context, obj interface{}, sourceEPs ...string) (*types.Responses, error) {
```

Comment:
Review should still take a variadic argument of opt... they just shouldn't be QueryOpts anymore (ReviewOpts?)

basically:

* Move QueryOpts to its own package separate from client & drivers (so both can import), rename to ReviewOpt
* Add a new option to specify source EPs
* Maybe separate out query opts that should be passed to drivers as their own class to avoid blurring abstractions.

The reason you don't want to use the variadic argument purely for EPs is that then there is nowhere to make backwards-compatible changes.